www/index.html

```diff
@@ -79,7 +79,7 @@
                         <div class="row">
                             <span class="col-10">
                                 <span class="input-group">
-                                    <input type="search" id="prefix" placeholder="Search..." class="form-control" />
+                                    <input type="search" id="prefix" placeholder="Search..." class="form-control" onclick = "this.focus()" />
```

Comment:
@KaranKaira Due to restrictive Content Security Policies in some contexts, we tyr to avoid using inline `on`-style events in HTML. Instead, you should add a click event listener to this input in `app.js`. I can't remember, but there may be such an event already defined in the code, in which case you can use it. On the other hand, I'm not sure why you want to focus the prefix on click, when a click should focus it anyway, without any additional code?

Reply:
hi @Jaifroid ,  Thanks for your review.
<blockquote> 
On the other hand, I'm not sure why you want to focus the prefix on click, when a click should focus it anyway, without any additional code?
</blockquote>
I did this to be 100%sure.
I will remove it now . 

Kindly see now.